Makefile

```diff
@@ -40,6 +40,10 @@ cluster-build:
 .PHONY: generate
 generate: crd update
 
+.PHONY: update
+update:
+	hack/update-generated-crd.sh
+
```

Comment:
Is this needed? In the current master branch, `update` comes from `vendor/github.com/openshift/build-machinery-go/make/targets/openshift/operator/profile-manifests.mk`, via the `$(call add-profile-manifests,...)` call on line 26

Reply:
Other repos use `make update` to generate code, and it was broken in this Makefile.  Right now if you run `make update` there is no error, but nothing happens. It's not strictly needed, but I'd like to be able to rely on `make update`.